modules/custom_operations/user_ie_extensions/tokenizer/python/README.md

```diff
@@ -57,6 +57,26 @@ cd user_ie_extensions/tokenizer/python/tests/
 pytest .
 ```
 
+### C++ Installation
+
+You can use converted tokenizers in C++ pipelines with prebuild binaries.
+
+1. Download OpenVINO archive distribution for your OS from [here](https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/download.html) and extract the archive.
+2. Download OpenVINO Tokenizers prebuild libraries from [here](storage.openvinotoolkit.org/repositories/openvino_tokenizers/packages/). To ensure compatibility first three numbers of OpenVINO Tokenizers version should match OpenVINO version and OS. 
+3. Extract OpenVINO Tokenizers archive into OpenVINO installation directory:
+    - Windows: `<openvino_dir>\runtime\bin\intel64\Release\`
+    - MacOS_x86: `<openvino_dir>/runtime/lib/intel64/Release`
+    - MacOS_arm64: `<openvino_dir>/runtime/lib/arm64/Release/`
+    - Linux_x86: `<openvino_dir>/runtime/lib/intel64/`
+    - Linux_arm64: `<openvino_dir>/runtime/lib/aarch64/`
+
+After that you can add binary extension in the code with:
+- `core.add_extension("openvino_tokenizers.dll")` for Windows
+- `core.add_extension("libopenvino_tokenizers.dylib")` for MacOS
+- `core.add_extension("libopenvino_tokenizers.so")` for Linux
```

Comment:
let's add on final names for 2023.3 before merge
https://github.com/openvinotoolkit/openvino_contrib/pull/837